packages/flutter/lib/src/rendering/sliver_fill.dart

```diff
@@ -59,69 +59,90 @@ class RenderSliverFillViewport extends RenderSliverFixedExtentBoxAdaptor {
   }
 }
 
-/// A sliver that contains a single box child that fills the remaining space in
-/// the viewport.
+/// A sliver that contains a single box child that contains a scrollable and
+/// fills the remaining space in the viewport.
```

Comment:
Is this correct? Doesn't it always grow to fill the entire viewport? (not just the "remaining" space)

Reply:
Thanks!